internal/terraform/aws/rds_cluster_instance.go

```diff
@@ -0,0 +1,39 @@
+package aws
+
+import (
+	"fmt"
+	"infracost/pkg/resource"
+)
+
+func NewRDSClusterInstance(address string, region string, rawValues map[string]interface{}) resource.Resource {
```

Comment:
Yeah I actually tried to follow the best practices in https://golang.org/doc/effective_go.html
I thought we would also rename all other acronym names to Uppercased, so I put it already like this. E.g. we have used `Gb` Acronym in our code which by we are referring to GB. It can mislead people to think it is Gigabit. Should I change it to `Rds` and let the renaming (if we agreed on) to happen in refactoring? or should I leave it like this?

Reply:
done

Reply:
worth mentioning dynamoDB has also this uppercase naming (variables such as `dynamoDBRCUQuantit` or `NewDynamoDBTable` (instead of `NewDynamoDbTable`) Should I change them as well to make them consistent?